client/package.json

```diff
@@ -24,9 +24,9 @@
     "@angular/platform-browser": "^17.1.2",
     "@angular/platform-browser-dynamic": "^17.1.2",
     "@angular/router": "^17.1.2",
-    "rxjs": "~7.8.0",
+    "rxjs": "^7.8.0",
```

Comment:
In lab 3, you have  "rxjs": "~7.8.1", but this is the only mismatch I noticed between lab 3 and here. I'm not sure what your intention is there.

Reply:
Thanks for asking about this! I think the "right" answer is to use the `^` instead of `~`, so what we probably want to do is change Lab 3.

Reply:
And, do you want the 7.8.1 or 7.8.0? Here, it's 7.8.0.